app/uk/gov/hmrc/soletraderidentificationfrontend/views/capture_sautr_page.scala.html

```diff
@@ -26,49 +26,45 @@
         govukInput: GovukInput,
         govukButton: GovukButton,
         formWithCSRF: FormWithCSRF,
+        h1: h1,
+        h2: h2,
         p: p,
         details: details,
         inputText: inputText,
         errorSummary: errorSummary
 )
-
 @(firstName: String, journeyId: String, pageConfig: PageConfig, formAction: Call, form: Form[String])(implicit request: RequestHeader, messages: Messages, appConfig: AppConfig)
 
 @layout(pageConfig, title = title("sa-utr.title", form)) {
 
     @errorSummary(form.errors)
 
+    @h1(messages("sa-utr.heading_1", firstName))
+
+    @p {@messages("sa-utr.line_1")}
+
+    @p {<a href="https://www.gov.uk/find-utr-number" class="govuk-link" rel="noreferrer noopener" target="_blank">@messages("sa-utr.line_2")</a>}
```

Comment:
Can you use the hmrclinkhelper for this e.g.:
https://github.com/hmrc/incorporated-entity-identification-frontend/blob/main/app/uk/gov/hmrc/incorporatedentityidentificationfrontend/views/capture_ctutr_page.scala.html#L46